### ISSUE TITLE
My hand is missing after installing the mod

### ISSUE THREAD

**Issue by ScienceDrummer** [Tommsy64/SmartMoving#8]
When I first installed the mod, I had noticed my hand was a little further back than usual. Then after switching around my tools, I noticed it completely disappeared.
![2018-05-26_15 50 28](https://user-images.githubusercontent.com/31962430/40888635-8eb1338a-671f-11e8-8d16-9b42ec438f91.png)
![2018-05-26_15 49 32_2](https://user-images.githubusercontent.com/31962430/40888636-9667e7b8-671f-11e8-9925-9540bd8e27df.png)
![2018-05-26_15 49 26](https://user-images.githubusercontent.com/31962430/40888637-9689b78a-671f-11e8-8474-9af989c38bbc.png)
![2018-05-26_15 49 31](https://user-images.githubusercontent.com/31962430/40888638-969bde74-671f-11e8-8cec-178d39166993.png)
![2018-05-26_15 49 31_2](https://user-images.githubusercontent.com/31962430/40888639-96af6e12-671f-11e8-80f5-7db2eb309c98.png)
![2018-05-26_15 49 31_3](https://user-images.githubusercontent.com/31962430/40888640-96ce03c2-671f-11e8-925d-b162eb5dc474.png)
![2018-05-26_15 49 32](https://user-images.githubusercontent.com/31962430/40888641-96dfb518-671f-11e8-8f81-544f4651717b.png)

**Comment by Tommsy64** [comment]
I've noticed the same thing; I have no idea what may be the exact cause of it. Switching to 3rd person and then back to 1st person seems to bring back the hand.

**Comment by ScienceDrummer** [comment]
Yeah, I forgot to mention that in the post about switching to F5 being able to fix it somewhat.
